ci/tests/webapp/validate-embedded-webapp.sh

```diff
@@ -6,6 +6,64 @@ gradleBuildOptions="--build-cache --configure-on-demand --no-daemon "
 
 webAppServerType="$1"
 
+createConfig() {
+  configDir=$1
+  echo "Creating config in ${configDir}"
+  mkdir -p ${configDir}
+  cat > ${configDir}/cas.properties <<EOF
```

Comment:
This is fine, but you didn't have to. Properties can always be passed as command-line arguments, such as `--management.endpoint.info.enabled=true`